app/marketing/views.py

```diff
@@ -122,6 +122,11 @@ def privacy_settings(request):
         if profile:
             profile.suppress_leaderboard = bool(request.POST.get('suppress_leaderboard', False))
             profile.hide_profile = bool(request.POST.get('hide_profile', False))
+            if profile.alumni:
```

Comment:
We'll want to modify the query on `profile` in `settings_helper_get_auth()` to `prefetch_related('alumni')`

Reply:
updated